<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleAdvertWidget</class>
 <widget class="QWidget" name="SaleAdvertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaleAdvertWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="508,292">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>508</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
	background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="98,97,399">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_3 {
	border: 1px solid #ededed;
}</string>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>25</y>
               <width>61</width>
               <height>20</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(102, 102, 102);
</string>
             </property>
             <property name="text">
              <string>商品名称</string>
             </property>
            </widget>
            <widget class="QLabel" name="goodsNameValueLabel">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>45</y>
               <width>221</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font:18px rgb(51, 51, 51);
font: bold;
font-family:&quot;微软雅黑&quot;;
</string>
             </property>
             <property name="text">
              <string>超级无敌尊贵水果大拼盘</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_6 {
	border: 1px solid #ededed;
}</string>
            </property>
            <widget class="CustomerPlatformLabel" name="platformValueLabel">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>37</y>
               <width>131</width>
               <height>34</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">/*font:18px;
font-family:&quot;微软雅黑&quot;;*/

color: rgb(116,163,93);
</string>
             </property>
             <property name="text">
              <string>-11.922</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>18</y>
               <width>54</width>
               <height>12</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(102, 102, 102);</string>
             </property>
             <property name="text">
              <string>台秤</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_4 {
	border: 1px solid #ededed;
}</string>
            </property>
            <widget class="QLabel" name="goodsPriceValueLabel">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>38</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 30px rgb(51, 51, 51);
font:  bold;
font-family:&quot;微软雅黑&quot;;</string>
             </property>
             <property name="text">
              <string>￥5.00</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_10">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>15</y>
               <width>54</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(102, 102, 102);</string>
             </property>
             <property name="text">
              <string>单价</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_7 {
	border: 1px solid #ededed;
}</string>
            </property>
            <widget class="QLabel" name="goodsNumValueLabel">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>38</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>-1</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font:30px rgb(51, 51, 51);
font:  bold;
font-family:&quot;微软雅黑&quot;;</string>
             </property>
             <property name="text">
              <string>1.126</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>15</y>
               <width>54</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(102, 102, 102);</string>
             </property>
             <property name="text">
              <string>数量</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_8 {
	border: 1px solid #ededed;
}</string>
            </property>
            <widget class="QLabel" name="goodsAmtValueLabel">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>38</y>
               <width>147</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>-1</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font:30px;
font: bold;
font-family:&quot;微软雅黑&quot;;
color:rgb(255,129,56);
</string>
             </property>
             <property name="text">
              <string>￥99.999</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>15</y>
               <width>54</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(102, 102, 102);</string>
             </property>
             <property name="text">
              <string>金额</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="StandbyAdvertWidget" name="advertImageWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>399</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CustomerSaleWidget" name="CustomerSaleListWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>292</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <zorder>layoutWidget</zorder>
    </widget>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder>CustomerSaleListWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StandbyAdvertWidget</class>
   <extends>QWidget</extends>
   <header>StandbyAdvertWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomerPlatformLabel</class>
   <extends>QLabel</extends>
   <header location="global">CustomerPlatformLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomerSaleWidget</class>
   <extends>QWidget</extends>
   <header location="global">CustomerSaleWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
